kbase-extension/scss/partials/_custom.scss

```diff
@@ -368,5 +368,28 @@ div.code_cell {
 }
 
 .panel-title {
-  font-size: 16px;
+  @include font-h4;
+}
+
+/* collapsible Bootstrap toggle */
+[data-toggle=collapse] {
```

Comment:
Here is the collapsible toggle CSS; it doesn't need to have the extra qualifying classes that it had elsewhere in the CSS files as the important part is the `data-toggle="collapse"` data attribute.